### PR TITLE
Branch nodeboxes

### PR DESCRIPTION
**Pull request by shadowshrig**
Made selection boxes for most nodebox nodes. some left with out because they looked better that way and likely some were missed

### CODE REVIEW

File: mods/furnature/wood_and_metal.lua [hunk 1]
```diff
@@ -45,6 +45,12 @@ qtcore.for_all_materials("wood", function (fields_wood)
                             { -1/16, -4/16, -6/16, 1/16, -1/16, -5/16, },
                             },
                             },
```
Comment (Fesmaster) [inline mods/furnature/wood_and_metal.lua]: This line should be indented one less - looks like a autoformat error.
(no, you didn't change it, but it should be fixed)

File: mods/qtcore/nodeboxes.lua [hunk 2]
```diff
@@ -74,4 +74,13 @@ qtcore.nb_dustpile = function()
 			{ -2/16, -8/16, -6/16, 2/16, -7/16, 6/16, },
 		}
 	}
+end
+
+qtcore.nb_dustpile_selection = function()
+	return{
```
Comment (Fesmaster) [inline mods/qtcore/nodeboxes.lua]: There should be a space between "return" and the bracket:
`return {`

**Review by Fesmaster — CHANGES_REQUESTED** [review]

File: mods/furnature/wood_and_metal.lua (hunk 1 above)
Comment (shadowshrig) [reply mods/furnature/wood_and_metal.lua]: done

File: mods/qtcore/nodeboxes.lua (hunk 2 above)
Comment (shadowshrig) [reply mods/qtcore/nodeboxes.lua]: was an issue with all in file, all fixed

**Review by Fesmaster — COMMENTED** [review]
Good to go (squash and merge)

**Pull request merged**